lib/mainloop-control.c

```diff
@@ -257,6 +257,21 @@ process_credentials(ControlConnection *cc, GString *command, gpointer user_data)
   control_connection_send_reply(cc, result);
 }
 
+static void
+show_files_present_in_config(ControlConnection *cc, GString *command, gpointer user_data)
```

Comment:
Not a change request, just an idea. There are unit tests in `lib/control/tests/test_control_cmds`, that might be extended with this functionality

Reply:
ok. I will look into that.

Reply:
I'll mark it unresolved until you could provide information on why not include test, or include test.

Reply:
Inside criterion tests, How can I add test paths in the configuration such that I can get those using ```list-files``` command and then compare the output string?

Reply:
The same way how grammar *does* it.

Reply:
@Kokan : I have gone through some of the criterion test files. But, I am not sure how to move forward. May you share any test case example?